.golangci.yaml

```diff
@@ -52,13 +53,20 @@ linters-settings:
       - shadow
       - fieldalignment
   goimports:
-    local-prefixes: "github.com/openfga/openfga"
+    local-prefixes: github.com/openfga/openfga
   importas:
     no-unaliased: true # Do not allow unaliased imports of aliased packages.
     no-extra-aliases: false # Do not allow non-required aliases.
     alias:
       - pkg: github.com/openfga/api/proto/openfga/v1
         alias: openfgav1
+  gci:
+    sections:
+      - standard # Standard section: captures all standard packages.
+      - default # Default section: contains all imports that could not be matched to another section type.
+      - prefix(github.com/openfga/api)  # Custom section: groups all imports with the specified Prefix.
+      - prefix(github.com/openfga/openfga)
+      - prefix(github.com/openfga/language)
```

Comment:
interesting, shouldn't these 2 be swapped based on order of the actual imports in the files?

Reply:
Yeah, i noticed after sending this PR that either the tool doesn't work as I expect, or I don't understand how to configure it properly.

Reply:
Removing `local-prefixes: "github.com/openfga/openfga"` seems to have fixed this.

Reply:
Not sure that we want to remove local-prefixes, I don't know what if anything goimports uses that for

Right now we're mixing local imports with other repos in the openfga org, which is weird imho. The standard approach would be that those other repos are considered part of the external group.

I'd suggest either going back to the standard 3-group layout; `standard, default, localmodule` or a 4-group layout; `standard, default, prefix(github.com/openfga), localmodule`

Reply:
goimports' local-prefixes seems to affect the order of the imports: https://golangci-lint.run/usage/linters/#goimports

> Reformat imports in autofix mode.

<img width="449" alt="image" src="https://github.com/user-attachments/assets/bc11e09a-88e8-442c-8944-a5b3fd200070">

> Right now we're mixing local imports with other repos in the openfga org, which is weird imho.

do you prefer to have

`openfga/api`, then
`openfga/language`, then
`openfga/openfga`?

Reply:
this config seems to do that:
```
  goimports:
    local-prefixes: "github.com/openfga/openfga"
  gci:
    sections:
      - standard # Standard section: captures all standard packages.
      - default # Default section: contains all imports that could not be matched to another section type.
      - prefix(github.com/openfga)  # Custom section: groups all imports with the specified Prefix.
      - localmodule  # Local section: contains all imports that are part of the same module.
```

Reply:
Added `localmodule`.